src/OutlineButton.js

```diff
@@ -3,22 +3,19 @@ import PropTypes from 'prop-types'
 import { color } from 'styled-system'
 import Button from './Button'
 
-const OutlineButton = styled(Button)`
+const OutlineButton = styled(Button) `
   background-color: transparent;
   border-width: 2px;
   border-style: solid;
-  border-color: currentcolor;
-  ${color}
+  color: ${props => props.theme.colors.blue};
+  border-color: ${props => props.theme.colors.blue};
+  
+  &:hover {
+    color: ${props => props.disabled ? props.theme.colors.blue : props.theme.colors.darkBlue};
+    border-color: ${props => props.disabled ? props.theme.colors.blue : props.theme.colors.darkBlue};
+    background-color: transparent;
```

Comment:
Does this need to also check for `disabled`?

Reply:
Oh I see it's transparent even not on hover. So I would remove this here. It doesn't do anything

Reply:
Talked to @Degron. This background-color needs to be there to override the hover background-color of the parent component (Button).